### PR TITLE
isbn-verifier: Fix typo in README

### PR DESCRIPTION
**Pull request by knepprath**
Used configlet to update README to the canonical problem description for isbn-verifier.

### CODE REVIEW

**Comment by cmccandless** [comment]
If we're fixing the README for this exercise anyway, perhaps we should conform to the [canonical problem description](https://github.com/exercism/problem-specifications/blob/master/exercises/isbn-verifier/description.md)?

**Comment by knepprath** [comment]
Good insight! I'm new to the project and have only been working with Python. I didn't realize there was a canonical description shared across languages. 

Maybe the README for every exercise should simply redirect to the canonical problem description. Seems like a fools errand trying to keep them all synchronized every time there is an update.

**Comment by cmccandless** [comment]
The intent behind maintaining separate copies is to allow each track to include track-specific information (how to run tests, implementation details, etc.). However, there is a [tool](github.com/exercism/configlet) that generates README files for a track using the canonical descriptions.

**Comment by knepprath** [comment]
@cmccandless, Thanks! This was a fun exercise for me to get deeper in to the project. 

I ran configlet on all the exercises and found 11 more that had diverged. What is the strategy for updating? Should I go ahead and do a single PR to update all of them?

**Comment by cmccandless** [comment]
Depends on what you mean by "diverged". I know of several that have a section on Exceptions which is unique to this track; these would remain as they are. 

I think it would be fine to do a single PR.

File: exercises/isbn-verifier/README.md [hunk 1]
```diff
@@ -1,43 +1,72 @@
...
+Now, it's even trickier since the check digit of an ISBN-10 may be 'X' (representing '10'). For instance `3-598-21507-X` is a valid ISBN-10.
+
+## Bonus tasks
+
+* Generate a valid ISBN-13 from the input ISBN-10 (and maybe verify it again with a derived verifier).
 
-The first 9 digits in the ISBN have to be between 0 and 9.
-The check digit can additionally be an 'X' to allow 10 to be a valid check digit as well.
+* Generate valid ISBN, maybe even from a given starting ISBN.
+## Exception messages
```
Comment (N-Parsons) [inline exercises/isbn-verifier/README.md]: Can you add a blank line before this heading?

Comment (knepprath) [reply exercises/isbn-verifier/README.md]: Done. 

On a side note. I did a spot check of a few of the description.md files in problem-specifications, and none of them end with a newline. So seems like all README's generated with the configlet will have similar formatting issue. Would we want the configlet to programmatically handle this?

**Comment by cmccandless** [comment]
https://github.com/exercism/python/commit/780fceea09cbd8e6b1205c18751b210a9c1207aa solved this issue, so I am going to close this PR.

@knepprath, regarding the whitespace issue you observed: as all README files in this track are generated as of #1179, editing individual README files by hand no longer makes sense. It would be best to determine which template file should include the additional blank line, create a PR to add it, and also regenerate all READMEs (same PR as template change). If you are interested in pursuing this issue, I would recommend taking a look at [this track's README template](https://github.com/exercism/python/blob/master/config/exercise_readme.go.tmpl).

**Comment by cmccandless** [comment]
@knepprath Actually, I believe the whitespace issue has been resolved already. If not, I am currently working on a PR to add another section to the track template, and will resolve it there.

**Pull request closed without merging**